templates/zerver/app/markdown_help.html

```diff
@@ -123,39 +121,28 @@
 
                     </tr>
                     <tr>
-                    <td class="preserve_spaces">```quote
+                    <td class="preserve_spaces apply_markdown">```quote
 Quoted block
 ```</td>
-                    <td class="rendered_markdown"><blockquote><p>Quoted block</p></blockquote></td>
+                    <td class="rendered_markdown"></td>
```

Comment:
I wonder if we could just have the rendering logic insert these `rendered_markdown` `<td>` elements, rather than having these empty blocks be required in each `tr` block?

Reply:
I could not find any other way than using empty blocks in each `tr` block. If you can suggest me any other method it would be very helpful to me.

Reply:
You should be able to do something like https://api.jquery.com/append/